app/api/images/schemas/image-post.req.schema.json

```diff
@@ -0,0 +1,166 @@
+{
+  "$id": "ImagePostRequest",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "properties": {
+    "iiif_link": {
+      "type": "string"
+    },
+    "is_published": {
+      "type": "boolean"
+    },
+    "original_id": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "caption": {
+      "type": [ "string", "null" ]
+    },
+    "collection_id": {
+      "type": "number"
+    },
+    "license": {
+      "type": "string"
+    },
+    "download_link": {
+      "type": [ "string", "null" ]
+    },
+    "link": {
+      "type": [ "string", "null" ]
+    },
+    "shop_link": {
+      "type": [ "string", "null" ]
+    },
+    "observation_enabled": {
+      "type": "boolean"
+    },
+    "correction_enabled": {
+      "type": "boolean"
+    },
+    "view_type": {
+      "oneOf": [
+        {
+          "type": "null"
+        },
```

Comment:
I don't think we should accept no view_type

Reply:
Should it be a required field too, then?

Reply:
yes, probably